deploy/helm/flightctl/values.kind.yaml

```diff
@@ -31,22 +31,19 @@ flightctl:
       pullPolicy: IfNotPresent
     nodePort: 15672 # this is also mapped in /hack/kind_cluster.yaml as an extraPortMapping
 
+flightctl-ui:
```

Comment:
Just `ui`? (Do we have any other than the flightctl one?

Reply:
Updated, it is now `ui` only